lib/csv/parser.rb

```diff
@@ -341,6 +343,8 @@ def prepare_regexp
       escaped_first_column_separator = Regexp.escape(@column_separator[0])
       escaped_row_separator = Regexp.escape(@row_separator)
       escaped_quote_character = Regexp.escape(@quote_character)
+      escaped_escape_character = Regexp.escape("\\".encode(@encoding))
```

Comment:
We can use "backslash" instead of "escape" because users can't customize the character for backslack escape.

Reply:
Fixed it. https://github.com/ruby/csv/pull/74/commits/754d72b2a6d4ad1baa360835b8677d7364e012ed